### DISCUSSION TITLE
useMove after opening a new window with global Window.open() function

### DISCUSSION THREAD

**Discussion by dbstraight** [adobe/react-spectrum#7217] [Ideas]
Because useMove registers event on the global window, it can't be used within a window opened with the global [Window.open](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) function.

Trying to use useMove on a component opened in a React portal in a new window.

**Comment by snowystinger** [comment]
Thanks for the discussion. I'm not sure what you're trying to accomplish. Could you provide a codesandbox or stackblitz with a minimal reproduction?
